_projects/gitmate_plugins.md

```diff
@@ -0,0 +1,57 @@
+---
+name: "Implement GitMate Plugins for coala"
+desc: "GitMate.io helps by automating tasks on GitHub and GitLab. There are new plugins that are specific to coala that should be implemented"
+requirements:
+  - "The participant should have at least one patch to [GitMate](https://gitlab.com/gitmate/open-source/gitmate-2) accepted."
+mentors: 
+  - sils
+  - nkprince007
+initiatives:
+  - GSoC
+difficulty: medium
+issues:
+  - "https://github.com/coala/projects/issues/488"
+tags:
+  - IGitt
+  - GitMate
+collaborating_projects:
+  - coala
+  - GitMate
+  - IGitt
+---
+
+[GitMate.io](https://gitmate.io/) is an automation tool for developers. While it works well, there are a number of plugins that could automate tasks to save maintainer time (for example automatically close PRs from master).
+
+[IGitt](https://igitt.gitmate.io/index.html) is a Git hoster abstraction 
+for python. It's an extensive framework that will help implement GitMate's plugins.
+
+To get started on GitMate, please follow 
+[our guide](http://docs.gitmate.io/Developers/Newcomers_Guide/).
+
+#### Plugin Ideas
+
+- Reject Pull Requests from master. Should include 
+instructions on how to fix their branch, fix their master, and resubmit a PR.
+- Fixes vs Closes automated checking.
+- A squash command to fix a PR with many messy commits. 
+Should link to the coala's git documentation for future reference
+- Automatically marking WIP if GitMate's coala plugin finds defects.
+  - Similarly marking PRs as WIP if CI tests fail
+- Auto-merger created to integrate rebase and ff plugins
+
+#### Milestones
+
+##### PREPARATION/BONDING
+- GitMate should automatically reject PRs from master
+
+##### Coding Phase 1
+- Fixes vs Closes automated checking should be implemented
```

Comment:
Also add a 'recheck' comment, for cases where the problem was unrelated to the PR, and has been fixed, just like has happened with your PR here.